packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -839,15 +841,33 @@ const CFDPasswordModal = ({
         if (!category && !type) return '';
 
         const category_label = category === 'real' ? localize('real') : localize('demo');
-        const type_label =
-            getMtCompanies(show_eu_related_content)[category as keyof TMtCompanies][
-                type as keyof TMtCompanies['demo' | 'real']
-            ].short_title;
-        const deriv_x_type_label =
-            getDxCompanies()[category as keyof TDxCompanies][type as keyof TDxCompanies['demo' | 'real']].short_title;
+        let type_label = '';
+        switch (platform) {
+            case CFD_PLATFORMS.MT5:
+                type_label =
+                    getMtCompanies(show_eu_related_content)[category as keyof TMtCompanies][
+                        type as keyof TMtCompanies['demo' | 'real']
+                    ].short_title;
+                break;
+            case CFD_PLATFORMS.DXTRADE:
+                type_label =
+                    getDxCompanies()[category as keyof TDxCompanies][type as keyof TDxCompanies['demo' | 'real']]
+                        .short_title;
+                break;
+            case CFD_PLATFORMS.DERIVEZ:
+                type_label =
+                    getDerivezCompanies()[category as keyof TDerivezCompanies][
+                        type as keyof TDerivezCompanies['demo' | 'real']
+                    ].short_title;
+                break;
+            default:
+                type_label = '';
+                break;
+        }
+
         const jurisdiction_label =
             jurisdiction_selected_shortcode && getFormattedJurisdictionCode(jurisdiction_selected_shortcode);
-        const mt5_platform_label = jurisdiction_selected_shortcode !== Jurisdiction.MALTA_INVEST ? 'MT5' : '';
+        const mt5_platform_label = jurisdiction_selected_shortcode !== Jurisdiction.MALTA_INVEST ? 'Deriv MT5' : '';
```

Comment:
@shontzu-deriv as per as prod, it should be MT5, not Deriv MT5
<img width="470" alt="image" src="https://github.com/binary-com/deriv-app/assets/82078941/8e1cd57a-4015-4415-aab8-eceb61a3a2f5">